### PR TITLE
feat: Adds assets used in the onboarding pages

### PR DESCRIPTION
**Pull request by nithindavid**
## Description
Adds SVG files and transalations needed for onboarding view

### CODE REVIEW

File: app/javascript/dashboard/i18n/locale/en/signup.json [hunk 1]
```diff
@@ -40,5 +40,88 @@
...
+        "LABEL": "Phone number (Optional)",
+        "PLACEHOLDER": "Enter your phone number"
+      },
+      "SIGNATURE": {
+        "LABEL": "Signature",
+        "PLACEHOLDER": "Add a signature for your messages. You can set this later in your settings."
+      }
+    },
+    "COMPANY": {
+      "PAGE_TITLE": "Hello %{userName},\nwe are are almost done.",
```
Comment (iamsivin) [inline app/javascript/dashboard/i18n/locale/en/signup.json:78]: @nithindavid What's the **nwe** word in the PAGE_TITLE?

**Review by iamsivin — COMMENTED** [review]
@nithindavid Added a small comment, please take a look.

File: app/javascript/dashboard/i18n/locale/en/signup.json (hunk 1 above)
Comment (nithindavid) [reply app/javascript/dashboard/i18n/locale/en/signup.json:78]: `\n` to break the sentence onto the next line.

**Review by iamsivin — APPROVED** [review]

File: app/javascript/dashboard/i18n/locale/en/signup.json (hunk 1 above)
Comment (iamsivin) [reply app/javascript/dashboard/i18n/locale/en/signup.json:78]: @nithindavid There are 2 **"are"** in this sentence.

Comment (nithindavid) [inline app/javascript/dashboard/i18n/locale/en/signup.json:78]:
```suggestion
      "PAGE_TITLE": "Hello %{userName},\nwe are almost done.",
```

**Review by iamsivin — APPROVED** [review]

**Pull request merged**